web/src/styles/operator.css

```
@@ -0,0 +1,595 @@
+}
+.op-led.is-off {
+  --led-color: var(--text-tertiary);
+}
+.op-led.is-info {
+  --led-color: #3b82f6;
+}
+
+.op-led.is-on {
+  animation: opLedPulse 2.8s ease-in-out infinite;
+}
+@keyframes opLedPulse {
```

Stylelint (17.12.0)

[error] Expected keyframe name "opLedPulse" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)